app/forms/support_interface/application_forms/reinstate_declined_offer_form.rb

```diff
@@ -0,0 +1,18 @@
+module SupportInterface
+  module ApplicationForms
+    class ReinstateDeclinedOfferForm
+      include ActiveModel::Model
+
+      attr_accessor :accept_guidance, :status, :audit_comment_ticket
+
+      validates :accept_guidance, :audit_comment_ticket, presence: true
+      validates :audit_comment_ticket, format: { with: /\A((http|https):\/\/)?(www.)?becomingateacher.zendesk.com\/agent\/tickets\// }
+
+      def save(course_choice)
```

Comment:
To fix the bug with the checkbox being deselected we can use 

`        self.accept_guidance = ActiveModel::Type::Boolean.new.cast(accept_guidance)
`

here. unfortunately the form doesn't know that 'true' == true when it renders on a failed save